### ISSUE TITLE
Color problems with "Call to action with grid layout with products and link"

### ISSUE THREAD

**Issue by carolinan** [WordPress/twentytwentyfive#470]
**Description**

In some of the style variations, the text color inside the cover block is not visible enough.

**Step-by-step reproduction instructions**
Select the "Twilight" style variation.
Insert a "Call to action with grid layout with products and link" pattern.
Confirm that the text between the images has a color contrast that is too low against the background:

![image](https://github.com/user-attachments/assets/5c2062ce-9891-48c2-8d6f-a5623a6f73f2)

Repeat the same for "Evening":
![image](https://github.com/user-attachments/assets/9c199a6e-11b8-417a-a99f-1bbeea433a9d)

And for Sunrise:
![image](https://github.com/user-attachments/assets/cfd9d178-c093-48a1-b05a-f0f0b2de3ad6)

In Twilight, it is kind of visible but does not live up to the text-to-background color contrast requirement. The ratio is  2.26:1.
![image](https://github.com/user-attachments/assets/de8c2085-feff-434f-85e5-9c49146e9622)
